main.go

```diff
@@ -15,190 +15,296 @@ import (
 )
 
 var (
-	configFile  = flag.String("config.file", "", "Path to config file.")
-	metricsPath = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+	addr           string
```

Comment:
We use this style of flag definition in Prometheus:

https://github.com/prometheus/prometheus/blob/master/main.go#L51-L76

(define the variable and flag in one line)